modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -559,6 +563,17 @@ public boolean stopRaftNodes(ReplicationGroupId groupId) {
         });
     }
 
+    @Override
+    public void destroyRaftNodeStorages(RaftNodeId nodeId, RaftGroupOptions groupOptions) {
```

Comment:
Do you think maybe we should try to delete all directories even if one of them is not deleted? 
Do something like:
```
IgniteUtils.closeAll(
    () -> groupOptions.getLogStorageFactory().destroyLogStorage(uri),
    () -> IgniteUtils.deleteIfExists(serverDataPath)
);
```

Reply:
Implemented it a bit differently to avoid the necessity to explicitly handle exceptions thrown by `closeAll()`